pkg/types/cluster.go

```diff
@@ -98,5 +116,5 @@ type ClusterPort struct {
 	ExposedPorts []ClusterExposedPort `json:"exposed_ports"`
 	CreatedAt    time.Time            `json:"created_at"`
 	Hostname     string               `json:"hostname"`
-	State        AddOnState           `json:"state"`
+	State        ClusterPortState     `json:"state"`
```

Comment:
Shouldn't this be the same as `ClusterAddonState`?

Reply:
I am making this match the API (cc @marccampbell ) https://github.com/replicatedhq/vandoor/blob/731095cf03709f399b558446c3654385ed3cdf68/vendor-api/cluster/types/types.go#L24-L29